lib/gamnit/network/server.nit

```diff
@@ -39,7 +39,7 @@
 #
 #     # `accept_clients` in non-blocking,
 #     # sleep before tying again, or do something else.
-#     nanosleep(0, 50000000)
+#     0.5.sleep
```

Comment:
I avoid superfluous parentheses when I can. And users of this module should understand how to call a method on a float, it's useful when writing games.